components/pages/acknowledgements/PageContent.tsx

```diff
@@ -27,7 +27,16 @@ import StyledLink from '#components/Link';
 import defaultTheme from '#components/theme';
 
 import Individuals from './Individuals';
-import Policy from './Policy';
+import {
+	iMicroSeqHeader,
+	iMicroSeqContent,
+	virusSeqContent,
+	virusSeqHeader,
+	Policy,
+	iMicroSeqAnchor,
+	virusSeqAnchor,
+} from './Policy';
+import Contributors from './Contributors';
 
 const PageContent = (): ReactElement => {
 	const theme: typeof defaultTheme = useTheme();
```

Comment:
theme is inferred correctly

Reply:
Defined this way throughout the repo
https://github.com/virusseq/portal-ui/pull/491